### PR TITLE
Remove `cra-kitchen-sink` example

### PR DESCRIPTION
**Pull request by tmeasday**
This storybook seemed to mainly just have some react rendering testing stories, which were either already created or not actually useful at all (we never looked at them, AFAIK).

### CODE REVIEW

File: code/examples/cra-kitchen-sink/src/stories/Lifecycle.stories.js [hunk 1]
```diff
@@ -1,8 +0,0 @@
-import React from 'react';
-import LifecycleLogger from '../components/LifecycleLogger';
-
-export default {
-  title: 'Lifecycle',
-};
-
-export const Logging = () => <LifecycleLogger />;
```
Comment (tmeasday) [inline code/examples/cra-kitchen-sink/src/stories/Lifecycle.stories.js:8]: You needed to check the browser console to see these, and half the methods weren't called anyway. Not sure there was a lot of benefit. If we see problems with react lifecycle, let's create a targetted story with a `play()` function to verify things work right.

File: code/examples/cra-kitchen-sink/src/stories/fast-refresh.stories.js [hunk 2]
```diff
@@ -1,9 +0,0 @@
-import React from 'react';
-import { FastRefreshExample } from '../components/FastRefreshExample';
-
-export default {
-  title: 'React Fast Refresh',
-  component: FastRefreshExample,
-};
-
-export const Default = () => <FastRefreshExample />;
```
Comment (tmeasday) [inline code/examples/cra-kitchen-sink/src/stories/fast-refresh.stories.js:9]: I've no idea what this tested, the component is really generic.

File: code/examples/cra-kitchen-sink/src/stories/force-rerender.stories.js [hunk 3]
```diff
@@ -1,19 +0,0 @@
-import React from 'react';
-import { forceReRender } from '@storybook/react';
-import { Button } from '../components/react-demo';
-
-let count = 0;
-const increment = () => {
-  count += 1;
-  forceReRender();
```
Comment (tmeasday) [inline code/examples/cra-kitchen-sink/src/stories/force-rerender.stories.js:8]: We have a story that triggers this event and verifies it.

File: code/examples/cra-kitchen-sink/src/stories/long-description.stories.js [hunk 4]
```diff
@@ -1,9 +0,0 @@
-import React from 'react';
-import { Button } from '../components/react-demo';
-
-export default {
-  title: 'Some really long story kind description',
-};
-
-export const Story1 = () => <Button>Hello Button</Button>;
-Story1.storyName = 'with text';
```
Comment (tmeasday) [inline code/examples/cra-kitchen-sink/src/stories/long-description.stories.js:9]: This is a UI unit test style story, 🤷

**Review by ndelangen — APPROVED** [review]

**Pull request merged**